assets/js/components/Dashboard/AlertBanner.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ComponentType } from "react";
+import { Alert } from "../../models/alert";
+import { ArrowRepeat } from "react-bootstrap-icons";
+import { formatEffect, translateRouteID } from "../../util";
+import { useParams } from "react-router-dom";
+
+interface Props {
+  alert: Alert;
+  isClosedAlert: boolean;
```

Comment:
Definitely a nit but does this need to be prop? We could just do the check on line 39 that we do on line 134 in Dashboard.tsx, I think

Reply:
Good call. I updated the logic a bit so that the `AlertBanner` takes the whole `bannerAlert` as a prop.